templates/event/show.html.twig

```diff
@@ -53,5 +53,20 @@
         {% endfor %}
         </ul>
         {% endif %}
-    </div>
+        {% if isParticipant %}
```

Comment:
looks like:

    {% if event.hasParticipant(app.user) %}

does'nt it?

Reply:
Totally, gonna change this